src/py/flwr/server/strategy/fedadagrad.py

```diff
@@ -85,12 +85,8 @@ def __init__(
                 during validation. Defaults to 2.
             min_available_clients (int, optional): Minimum number of total
                 clients in the system. Defaults to 2.
-            evaluate_fn : Optional[
-                Callable[
-                    [int, NDArrays, Dict[str, Scalar]],
-                    Optional[Tuple[float, Dict[str, Scalar]]]
-                ]
-            ]: Function used for validation. Defaults to None.
+            evaluate_fn : Optional[Callable[[int, NDArrays, Dict[str, Scalar]], Optional[Tuple[float, Dict[str, Scalar]]]]]
```

Comment:
Why do we need this?

Reply:
Current rendering is broken see here:
<img width="791" alt="Screenshot 2022-10-05 at 13 36 31" src="https://user-images.githubusercontent.com/423981/194051729-569eab34-c5d0-4c56-9f48-3aae7c480186.png">

Reply:
vs.
<img width="823" alt="Screenshot 2022-10-05 at 13 42 21" src="https://user-images.githubusercontent.com/423981/194052473-c8ac0150-5fb6-4513-990e-23e7311238f0.png">

Reply:
@tanertopal before we do this, can we format the docstrings in question using NumPy style (the format that we try to use everywhere)?